samcli/lib/providers/cfn_api_provider.py

```diff
@@ -75,6 +95,208 @@ def extract_resources(self, stacks: List[Stack], collector: ApiCollector, cwd: O
                 if resource_type == AWS_APIGATEWAY_V2_STAGE:
                     self._extract_cfn_gateway_v2_stage(resources, resource, collector)
 
+                if resource_type == AWS_APIGATEWAY_V2_AUTHORIZER:
+                    self._extract_cfn_gateway_v2_authorizer(logical_id, resource, collector)
+
+    @staticmethod
+    def _extract_cloud_formation_authorizer(logical_id: str, resource: dict, collector: ApiCollector) -> None:
+        """
+        Extract Authorizers from AWS::ApiGateway::Authorizer and add them to the collector.
+
+        Parameters
+        ----------
+        logical_id: str
+            The logical ID of the Authorizer
+        resource: dict
+            The attributes for the Authorizer
+        collector: ApiCollector
+            ApiCollector to save Authorizers into
+        """
+        properties = resource.get("Properties", {})
+        authorizer_type = properties.get(CfnApiProvider._AUTHORIZER_TYPE, "").lower()
```

Comment:
Just curious, why we are calling `lower` here?

Reply:
When defining the type in `Properties`, the expected values are `TOKEN` and `REQUEST`, but if defined in swagger, then its `token` or `request`. To make things consistent, I elected to make things lowercase once it makes its way into SAM CLI.

Reply:
Oh, a good point here would be to also consider validating this behaviour.